io/pom.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Both the SDK and the connectors are specific to platform (JVM), not really language. The platform has one global namespace that you put things in and anything else on the platform can use them.